bin/test_runner.rb

```diff
@@ -58,21 +71,43 @@ def is_up?
   end
 
   def get_line
```

Comment:
We only use this method once, and since it's only used to create a string to print out, we should remove this to simplify things.